app/models/chargeSummary/ChargeSummaryViewModel.scala

```diff
@@ -35,27 +34,21 @@ case class ChargeSummaryViewModel(
                                    paymentAllocationEnabled: Boolean,
                                    latePaymentInterestCharge: Boolean,
                                    codingOutEnabled: Boolean,
+                                   reviewAndReconcileEnabled: Boolean,
```

Comment:
not sure if we adding FS to the corresponding models, we can keep it for the time been, but I guess in general we keep FS separately from the view model ?